settings/data/utils/file.ts

```
@@ -1,59 +1,84 @@
-import fs from 'fs';
-import { handleError } from './logger';
+import { existsSync, readFileSync, writeFileSync } from 'fs';
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)